ansible_base/lib/dynamic_config/settings_logic.py

```diff
@@ -283,6 +283,17 @@ def get_dab_settings(
 
         dab_data['ALLOW_OAUTH2_FOR_EXTERNAL_USERS'] = False
 
+        dab_data['ANSIBLE_BASE_OAUTH2_PROVIDER_PERMISSIONS_CHECK_IGNORED_VIEWS'] = [
```

Comment:
Add `if ansible_base.api_docs in installed_apps:`. Also. if drf_spectacular has its own settings for permissions classes we could add a second check to make sure those aren't set and then add these entries.

Reply:
Leaving the addition of the ignore up to the check users, since by default api_docs uses AllowAny as far as I'm aware.